src/component/page/deltakerliste-detaljer/deltaker-oversikt/deltaker-tabell/Rad.tsx

```diff
@@ -82,10 +83,12 @@ export const Rad = (props: RadProps): React.ReactElement<RadProps> => {
 		<Table.Row key={id}>
 			<Table.DataCell className={ styles.smallText }>
 				<Link className={styles.brukersNavn} to={brukerDetaljerPageUrl(id, 'koordinator')} onClick={() => loggKlikk(klikkDeltakerRadOversikt)}>
-					{deltakerNavn}
+					{ adressebeskyttet ? 'Adressebeskyttet' : deltakerNavn }
```

Comment:
Jeg tror dette må gjøres i veileders tabell også, siden navn ikke sendes med til tabellvisningen der eller så nå vises bare et `,` 😅

Reply:
ja, lurer på om jeg kanskje bare skal merge de to oppgavene med koordinator og veieder?

Reply:
Åja, da så! 😅 Det er opp til deg, jeg vet ikke hvor stor endring det er for veileders del.